community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/NodeFirstRelationshipProcessor.java

```diff
@@ -52,15 +52,13 @@ public boolean process( NodeRecord node )
     {
         long nodeId = node.getId();
         long firstRel = nodeRelationshipLink.getFirstRel( nodeId, this );
-        if ( firstRel == -1 )
+        if ( firstRel != -1 )
         {
-            return false;
```

Comment:
We used to return false if the first rel was missing but now we always return true. I wasn't sure of the reason for that change in behaviour?